pygmy/rest/shorturl.py

```diff
@@ -44,7 +44,7 @@ def post(self):
         if errors:
             log.error('Error in the request payload %s', errors)
             if errors.get('long_url'):
-                errors.update({'error': errors.get('long_url')})
+                errors.update({'error': " ".join(errors.get('long_url'))})
```

Comment:
I think we can get rid of this. This is not required anymore after changes to the template.

Reply:
Oops. Sorry, i forgot to remove join.